### PR TITLE
timezone copy updates and focus TZ selector on cancel

### PR DESCRIPTION
**Pull request by krystophv**
Some copy updates and simple state additions to handle keeping track of/setting the focus state of the timezone selector when an upload is cancelled from the time check dialog.

Trello card: https://trello.com/c/ERwBfwRa

### CODE REVIEW

**Comment by krystophv** [comment]
[Update Device Time Check Copy](https://trello.com/c/ERwBfwRa/369-update-device-time-check-copy)

File: app/components/TimezoneDropdown.js [hunk 1]
```diff
@@ -25,6 +25,16 @@ var cx = require('classnames');
 var styles = require('../../styles/components/TimezoneDropdown.module.less');
 
 class TimezoneDropdown extends React.Component {
+  constructor(props) {
+    super(props);
```
Comment (cbwebdevelopment) [inline app/components/TimezoneDropdown.js]: Don't actually need to pass `props` to `super` here, but not hurting anything either.

File: app/components/TimezoneDropdown.js [hunk 2]
```diff
@@ -25,6 +25,16 @@ var cx = require('classnames');
...
+    this.timezoneSelect = null;
+
+    this.setTimezoneSelect = element => {
+      this.timezoneSelect = element;
+    };
+  }
+
   static propTypes = {
     onTimezoneChange: PropTypes.func.isRequired,
     selectorLabel: PropTypes.string.isRequired,
```
Comment (cbwebdevelopment) [inline app/components/TimezoneDropdown.js]: Should add `onBlur` propType declaration

Comment (cbwebdevelopment) [reply app/components/TimezoneDropdown.js]: also `isTimezoneFocused`

File: app/components/TimezoneDropdown.js (hunk 1 above)
Comment (cbwebdevelopment) [reply app/components/TimezoneDropdown.js]: Actually, NVM. I see that they recommend it in the React docs.
`Class components should always call the base constructor with props`

Currently, it works without it, but should leave it as per their recommendations.

**Pull request merged**